pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -133,7 +133,7 @@ def test_get_data_multiple_symbols(self):
 
     @pytest.mark.parametrize('adj_pr', [True, False])
     def test_get_data_null_as_missing_data(self, adj_pr):
-        #workaround as skip_on_exception decorator changes signature
```

Comment:
@bashtage    thanks for the code sample interesting usage of fixtures, wasn't able to run it successfully,  but I am no pytest expert just a casual user; also this approach has an additional level of indirection(a new fixture) and probably some advanced pytest knowledge but a signature-preserving decorator hopefully should work just like any other decorator, just put one line in front of a function